api/src/pcapi/alembic/versions/20230608T150904_17adb47d509c_create_table_product_whitelist.py

```diff
@@ -0,0 +1,42 @@
+"""create_table_product_whitelist
```

Comment:
Ce message doit être lisible, car il apparaît dans la sortie de la commande `alembic history` : 

```suggestion
"""Create "product_whitelist" table
```